MapboxNavigation/Resources/zh-Hans.lproj/Navigation.strings

```diff
@@ -0,0 +1,15 @@
+
+/* Class = "UIButton"; normalTitle = "End Navigation"; ObjectID = "5f2-dT-la4"; Note = "DO NOT TRANSLATE"; */
```

Comment:
These strings were not pulled from Transifex, right? The "DO NOT TRANSLATE" ones should be stripped out. Please update the strings in Transifex and do `tx pull -f` followed by running the `./convert_string_files.sh` from the script folder. For more information, see https://github.com/mapbox/mapbox-navigation-ios/blob/master/CONTRIBUTING.md#adding-or-updating-a-localization